src/main/scala/shell/microsemi/PolarFireEvalKitShell.scala

```diff
@@ -497,11 +497,10 @@ abstract class PolarFireEvalKitShell(implicit val p: Parameters) extends RawModu
   // DDR3 Subsystem Clocks
   //-----------------------------------------------------------------------
   val ddr3_clk_ccc = Module(new PolarFireCCC(
-		PolarFireCCCParameters(
-    	name = "ddr3_clk_ccc",
-    	pll_in_freq = 50,
-    	gl0Enabled = true,
-    	gl0_0_out_freq = 111.111)))
+    PLLParameters(
+      name = "ddr3_clk_ccc",
+      PLLInClockParameters(50),
+      Seq(PLLOutClockParameters(111.111)))))
```

Comment:
might want to check that these parameters are correct. This was just my pattern-match best guesses.

Reply:
Yup I will double check, but they look fine so far